### PR TITLE
Revamp navigation menu, close MISC-178

### PR DESCRIPTION
**Pull request by notdryft**
Motivation:

- Unclear navigation path
- Too much clicking to get anywhere

Modification:

- Added the navigation menu in the listing page
- Navigation now shows from root and not only the current section and
  children
- Menu will find the appropriate version and lock the reference section
  to this specific version, not showing the other possibilities
- Menu will expand to the current path and collapse the rest
- Make sure bootstrap.js is never imported as do everything manually

Result:

- Clearer navigation path and it is not easier to find content

### CODE REVIEW

**Comment by notdryft** [comment]
Screenshot_20211115_113941

File: assets/js/menu.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const menuCollapse = (event) => {
+  event.preventDefault();
+
+  event.currentTarget.setAttribute(
+    "aria-expanded",
+    event.currentTarget.getAttribute("aria-expanded") === "true"
+      ? "false"
+      : "true"
```
Comment (Isammoc) [inline assets/js/menu.ts:8]: not real boolean???

Comment (gpoulet) [reply assets/js/menu.ts:8]: getAttribute() returns a string or null

Comment (Isammoc) [reply assets/js/menu.ts:8]: sad :sob:

**Review by tpetillot — APPROVED** [review]

**Pull request merged**